kasa/httpclient.py

```diff
@@ -60,6 +67,12 @@ async def post(
 
         If the request is provided via the json parameter json will be returned.
         """
+        if self._wait_between_requests:
+            now = time.time()
+            gap = now - self._last_request_time
+            if gap < self._wait_between_requests:
+                await asyncio.sleep(self._wait_between_requests - gap)
```

Comment:
Should we zero `self._wait_between_requests` after the sleep?

Reply:
Once we know a device needs a wait between requests it should always wait. If we zero it then the device will error next time round.